### ISSUE TITLE
Request: Disable hotkeys when typing

### ISSUE THREAD

**Issue by Hotklou2404** [aequis/ModernCamera#1]
When typing in the chat or the console, pressing one of the keys that are used by ModernCamera activate's ModernCamera's functions. It'd be great if the hotkeys were disabled when typing.

**Comment by DiabloFan8675309** [comment]
Yes, please! This is the only reason I am still using the older version, driving me nuts.
